docs/content/latest/yugabyte-cloud/cloud-faq.md

```diff
@@ -73,6 +75,18 @@ If you want to continue testing YugabyteDB with more resource-intensive scenario
 
 Currently self-service migration is not supported. Contact [Yugabyte Support](https://support.yugabyte.com/hc/en-us/requests/new?ticket_form_id=360003113431) for help with migration.
 
+### What is the upgrade policy for clusters?
+
+Upgrades are automatically handled by Yugabyte. There are two types of upgrades:
+
+Cloud console
+: During a maintenance window, the Yugabyte Cloud console may be in read-only mode and not allow any edit changes. The upgrade has no impact on running clusters. Customers will be notified in advance of the maintenance schedule.
+
+Cluster (yugabyteDB) version upgrade
+: To keep up with the latest bug fixes, improvements, and security fixes, Yugabyte will upgrade your cluster to the [latest version](#what-version-of-yugabytedb-does-my-cluster-run-on). We will notify you of any upcoming upgrade schedule via email.
+: The database is upgraded to the latest release in the release track that was selected when the cluster was created (either edge or stable). Free clusters are always in the edge track.
```

Comment:
Add a link to "release track". I know it's pointing like four lines down right now, but it might point somewhere quite different, in the future...